enterprise/server/test/webdriver/invocation/invocation_test.go

```diff
@@ -2,18 +2,90 @@ package invocation_test
 
 import (
 	"context"
+	"flag"
 	"testing"
 
 	"github.com/buildbuddy-io/buildbuddy/enterprise/server/testutil/buildbuddy_enterprise"
+	"github.com/buildbuddy-io/buildbuddy/server/testutil/app"
 	"github.com/buildbuddy-io/buildbuddy/server/testutil/testbazel"
 	"github.com/buildbuddy-io/buildbuddy/server/testutil/webtester"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 )
 
-func TestAuthenticatedInvocation_LogUploadEnabled(t *testing.T) {
+var (
+	targetStr   = flag.String("webdriver_target", "local", "Target that should be tested by the webdriver.")
+	devSSOSlug  = flag.String("webdriver_dev_slug", "", "Dev SSO slug to login.")
```

Comment:
The plan is to save these SSO slugs as secrets in the github action that will do the release. Then after we deploy dev for example, we can run this webdriver test on the dev target with the dev SSO secret